workflows/workflows/mapping.wdl

```diff
@@ -10,14 +10,16 @@ workflow mapping {
         Array[File] index_files
         File metricsfile
         File? blacklist
+        Int? read_length = 75
```

Comment:
including the `read_length` in the subworkflow might not be necessary, because it is not provided in the main workflow.